lib/diagnostic.js

```diff
@@ -6,13 +6,16 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function (splitWords) {
+  let words = "Today Is The Day After Wednesday.";
+  let results = words.split(" ");
 };
 
 // 2. Write a function that takes a string argument
 // and returns an array of unique normalized words.
 const uniqueWords = function (/* */) {
+  let words = "Today Is The Day After Wednesday.";
+  let results = "words.normalizeWords[]";
```

Comment:
Anything in quotes is a string literal. In this line, you defined the variable `results` as "words.normalizeWords[]". What you want are the results from the function, the syntax of which is `let words = normalizeWords[words]`. Of course, this would only work if your `normalizeWords` function returned something.